Raptacon/Robot-2023
# Selector

## Types of changes


## Associated Issue(s)
issue #78 

## Steps to Test
Connect a controller and press B, then mess with the D-Pad and see the selection in shuffleboard.

## Changelog
A selector would be beneficial for changing between modes on the arm.

## Why?
I am making this change so we can switch between the different arm modes.
